package.nls.json

```diff
@@ -125,6 +125,8 @@
     "AWS.command.stepFunctions.publishStateMachine": "Publish state machine to Step Functions",
     "AWS.command.stepFunctions.previewStateMachine": "Render state machine graph",
     "AWS.command.viewLogStream": "View Log Stream...",
+    "AWS.command.ssmDocument.createDocumentFromTemplate": "Create a new local SSM Document from template",
```

Comment:
I am using `createDocumentFromTemplate` to distinguish the creating local and remote document experiences, because the API name for creating remote document is `createDocument`. Maybe `createLocalDocument` makes more sense.

Reply:
Okay, I suggest naming the command `createLocalDocument`, and changing the text to "Create a new SSM Document locally"

Reply:
OK